pkgs/applications/science/physics/sherpa/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "sherpa-${version}";
-  version = "2.2.5";
+  version = "2.2.6";
 
   src = fetchurl {
```

Comment:
Since they seem to have moved their website and code (as well as realeases) to GitLab I think it's worth changing to `fetchFromGitLab` to get these sources.

Reply:
@etu 
The gitlab release page [1] still points to hepforge for download, so I think this is fine for now. Also, I just checked that switching to fetching from git tag gives a slightly different source code than what is in the released tarball.

[1] https://gitlab.com/sherpa-team/sherpa/releases

Reply:
Reported the discrepancy upstream https://gitlab.com/sherpa-team/sherpa/issues/155

Reply:
@etu There was a reply in the upstream ticket. This is a funny situation, it seems like our diligence helped to find a buggy release. 👍

Reply:
@veprbl Hah, that's amazing :-)